.circleci/config.yml

```diff
@@ -61,6 +61,8 @@ jobs:
             # fallback to using the latest cache if no exact match is found
             - v1-dependencies-
 
+      - run: yarn lint:check
```

Comment:
This probably shouldn't be a part of the checkout. It should be its own job, and run between checkout and other `builds-*`